src/main/java/edu/cornell/kfs/module/ar/batch/service/impl/CuVerifyBillingFrequencyServiceImpl.java

```diff
@@ -0,0 +1,93 @@
+package edu.cornell.kfs.module.ar.batch.service.impl;
+
+import org.apache.commons.collections.CollectionUtils;
```

Comment:
This should use the collections4 version of CollectionUtils instead.

Reply:
updated